Lib/platform.py

```diff
@@ -409,7 +409,12 @@ def win32_ver(release='', version='', csd='', ptype=''):
 def _mac_ver_xml():
     fn = '/System/Library/CoreServices/SystemVersion.plist'
     if not os.path.exists(fn):
-        return None
+        if 'SDKROOT' in os.environ:
```

Comment:
This affects more than just the build. Why is this necessary?

Reply:
To install wheels when cross-compiling, this is needed to make sure wheels know what platform that it is targetting.
Setting `SDKROOT` is supported by clang and I thought python should support that too. I can remove it and figure out a way to monkeypatch it if you think it is too controversial.

Reply:
I'm -1 on this particular change, see also my latest comment in the main discussion.